src/pyfaf/storage/fixtures/__init__.py

```diff
@@ -451,7 +452,7 @@ def restore_lob_dir(self, url=None, cache=True):
                     bytes_so_far += len(chunk)
                     f.write(chunk)
 
-                    percent = float(bytes_so_far) / total_size
+                    percent = float(bytes_so_far) // total_size
```

Comment:
This should stay just /

Reply:
You are right, it should stay ```/``` because the first operator is ```float```.